unstructured_ingest/v2/processes/connectors/qdrant.py

```diff
@@ -0,0 +1,197 @@
+import asyncio
+import json
+import uuid
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Optional
+
+from pydantic import Field, Secret
+
+from unstructured_ingest.error import DestinationConnectionError, WriteError
+from unstructured_ingest.utils.data_prep import batch_generator, flatten_dict
+from unstructured_ingest.utils.dep_check import requires_dependencies
+from unstructured_ingest.v2.interfaces import (
+    AccessConfig,
+    ConnectionConfig,
+    FileData,
+    Uploader,
+    UploaderConfig,
+    UploadStager,
+    UploadStagerConfig,
+)
+from unstructured_ingest.v2.logger import logger
+from unstructured_ingest.v2.processes.connector_registry import DestinationRegistryEntry
+
+if TYPE_CHECKING:
+    from qdrant_client import AsyncQdrantClient
+
+
+CONNECTOR_TYPE = "qdrant"
+
+
+class QdrantAccessConfig(AccessConfig):
+    api_key: Optional[str] = Field(default=None, description="API Key")
+
+
+class QdrantConnectionConfig(ConnectionConfig):
+    collection_name: str = Field(description="Collection Name")
+    location: Optional[str] = Field(default=None, description="Location")
+    url: Optional[str] = Field(default=None, description="URL")
+    port: Optional[int] = Field(default=6333, description="Port")
+    grpc_port: Optional[int] = Field(default=6334, description="GRPC Port")
+    prefer_grpc: Optional[bool] = Field(default=False, description="Prefer GRCP")
+    https: Optional[bool] = Field(default=None, description="HTTPS")
```

Comment:
For these help descriptions, can we leverage better text pulled from the client sdk: [async_qdrant_client.py](https://github.com/qdrant/qdrant-client/blob/master/qdrant_client/async_qdrant_client.py#L59-L84)